ecommerce/managing_products/variants.rst

```diff
@@ -1,92 +1,110 @@
-==============================
-How to manage product variants
-==============================
+=======================
+Manage product variants
```

Comment:
Overall, the documentation looks a bit poor.
+ I really don't like the "list" view this documentation has. I would prefer something that looks more like a "continuous text". @jcs-odoo